# Verification
The point in this is to see if the config makes sense!

In [1]:
import numpy as np
import json

In [4]:
with open("agents.json", "r") as f:
    agents = json.load(f)

In [9]:
np.unique([a["actions"]["0"] for a in agents], return_counts=True)

(array(['59b954e3-da71-4ad5-91b6-b7a8a55ec84b',
        '78cfcc29-09ed-4122-bb7a-2e867bed82bc',
        '83908591-344f-4ff6-b018-03138f731a87',
        '98548711-2dc8-45f6-9998-c13fab43b6d4'], dtype='<U36'),
 array([155, 196,  49, 100]))

100 Drivers
49 Walkers
155 Cyclers
196 PTs

Maybe a bit unbalanced towards cyclists, but it doesn't matter too much here.

&#9989; Pass validation for toy model